tests/main/refresh/task.yaml

```diff
@@ -62,6 +62,7 @@ restore: |
         . $TESTSLIB/store.sh
         teardown_fake_store $BLOB_DIR
     fi
+    rm -f test-snapd-tools_*.snap
```

Comment:
I think this line should be part of `restore` in `tests/main/refresh-amend/task.yaml`